packages/core/src/list-transactions.ts

```diff
@@ -132,7 +136,7 @@ export async function listTransactions(
             lastTxIndex === networkInteraction.transactions.length - 1
           ),
           to: networkInteraction.to,
-          value: networkInteraction.value,
+          value: networkInteraction.value.toString(),
```

Comment:
Similarly I think the value of network interaction is a `bigint`, lets keep that when we are passing data around rather than letting a UI concern leak in here.